src/patterns/definitions.owl

```diff
@@ -0,0 +1,477 @@
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/AfPO_0000039> "A country in Western Africa with a population size of 0.77 million")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/AfPO_0000040> "A country in Central Africa with a population size of 4.95 millions")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/AfPO_0000041> "A country in Northern Africa with a population size of 0.6 million")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/AfPO_0000042> "A country in Northern Africa with a population size of 33.9 millions")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/AfPO_0000043> "A country in Northern Africa with a population size of 97 millions")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/AfPO_0000044> "A country in Eastern Africa with a population size of 37.3 millions")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/AfPO_0000045> "A country in Northern Africa with a population size of 40.9 millions")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/AfPO_0000046> "A country in Northern Africa with a population size of 6.65 millions")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/AfPO_0000047> "A country in Northern Africa with a population size of 11.4 millions")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/AfPO_0000048> "A country in Southern Africa with a population size of 2.214 millions")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/AfPO_0000049> "A country in Southern Africa with a population size of 2.48 millions")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/AfPO_0000050> "A country in Southern Africa with a population size of 1.46 millions")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/AfPO_0000051> "A country in Southern Africa with a population size of 54.84 millions")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/AfPO_0000052> "A country in Southern Africa with a population size of 1.95 million")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/AfPO_0000056> "A country in Nilotic Eastern with a population size of 0.9 millions")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/AfPO_0000057> "A country in Nilotic Eastern with a population size of 0.1 millions")
```

Comment:
Statements from here on down are nonsensical. These are not countries, they are populations, and the "in X" are population groups or categories, not geographical regions. We need different patterns for countries and populations.

Reply:
Thank you for reviewing the PR, Dani. I'll create another pattern for country.